docs/release-notes/2.41.0.md

```diff
@@ -0,0 +1,75 @@
+[PMM-12459](https://jira.percona.com/browse/PMM-12459) - The [pmm_dump client utility](https://docs.percona.com/pmm-dump-documentation/index.html) previously available as a standalone client for PMM Server is now readily accessible within the PMM user interface. 
+[PMM-12463](https://jira.percona.com/browse/PMM-12463) - [Telemetry] Collecting information about env variables usage in PMM
+
+## Improvements
+
+[PMM-11341](https://jira.percona.com/browse/PMM-11341) - PMM 2.41.0 introduces limit for Auto-discovery in PostgreSQL, a feature that dynamically discovers all databases in your PostgreSQL instance. Limiting Auto-discovery reduces connections and prevents high CPU and RAM usage caused by multiple databases.
+[PMM-12375](https://jira.percona.com/browse/PMM-12375) - Starting with PMM 2.41.0, each instance of a service gets a `version` attribute to the response of the endpoint that provides a list of services being monitored by PMM.
+[PMM-12422](https://jira.percona.com/browse/PMM-12422) - Starting with PMM 2.41.0, a new flag called `--expose-exporter-address` has been introduced. When you enable this flag and set the metrics mode to **push**, the node exporter for that agent will use the IP address `0.0.0.0`. This means that any IP address, either from a local system or from anywhere on the internet, can access it. If the flag is not enabled, the node exporter will use the address `127.0.0.1`, the default name for the computer you are working on, or the localhost.
+[PMM-12544](https://jira.percona.com/browse/PMM-12544) - Added deprecation notice to the DBaaS pages, to advise about the evolution of PMM DBaaS functionality into [Percona Everest](http://per.co.na/pmm-to-everest).
+[PMM-12549](https://jira.percona.com/browse/PMM-12549) - Added support for the new MongoDB version, which means you can use PMM to monitor MongoDB 7 databases.
+## Components upgrade
+
+- [PMM-12154](https://jira.percona.com/browse/PMM-12154) - postgres_exporter has been updated to version [0.14.0](https://github.com/prometheus-community/postgres_exporter/releases).
+- [PMM-12223](https://jira.percona.com/browse/PMM-12223) - Updated Clickhouse to v22.8.
+
+
+## Bugs Fixed
+
+- [PMM-4712](https://jira.percona.com/browse/PMM-4712) - We have addressed the issue where the [pprof](https://github.com/google/pprof) heap reports were missing. As a result, you can investigate out-of-memory issues more efficiently and precisely.
+- [PMM-12576](https://jira.percona.com/browse/PMM-12576) - After upgrading to PMM 2.40.0, changing the Admin user’s password from the terminal was not functioning. The issue has been resolved now.
+- [PMM-12626](https://jira.percona.com/browse/PMM-12626) - The GO version that we used to build PMM was updated to the latest version to mitigate potential vulnerabilities.
```

Comment:
Phrasing sugg:
The GO version that we used to build PMM was updated to the latest version to mitigate potential vulnerabilities.

to 

We updated the GO version used to build PMM to the latest version, to mitigate potential vulnerabilities.

Keep active voice.